### PR TITLE
398 update thyra readme to install and run thyra on testnet18

### PR DESCRIPTION
**Pull request by SlnPons**

### CODE REVIEW

File: INSTALLATION.md [hunk 1]
```diff
@@ -179,20 +179,15 @@ If you want to get down to business, you can start your [Massalian journey](http
...
-2. Type "cmd" 
-3. Right click on the "Open command prompt" icon and click on "Execute as an administrator"
-
-Now, copy the cmd below and paste it in the terminal you just opened.
+Follow the link below and download the file named "ThyraApp_windows-amd64.exe" on your computer. Then: 
+1. Open it by double clicking
+2. The script starts and here is what it does:
+** If you have Thyra already installed on your computer, it install Thyra's icon tray. 
+![windows_icontray_V0](https://user-images.githubusercontent.com/109611779/212294116-05e1dd37-ed3f-4e3e-b034-b02d782bc4ee.png)
+** If nothing is installed on your computer yet, it will install both.
```
Comment (gregLibert) [inline INSTALLATION.md:187]: It's not [the rendering](https://github.com/massalabs/thyra/blob/7b0218b8f38ae351872eb38df5f6a37278ac3823/INSTALLATION.md) you expect I think.

**Review by gregLibert — CHANGES_REQUESTED** [review]
:no_good:

File: README.md [hunk 2]
```diff
@@ -121,7 +121,7 @@ Accepts a URL, an IP address or one of the following values :
 - LOCALHOST : Expect Massa's network to be hosted at 127.0.0.1
 
 To use this option with a constant, you have to execute :
-`thyra-server node-server=LABNET`
+`thyra-server -node-server=LABNET`
```
[the remark this answers is not in the document]
Comment (SlnPons) [reply README.md:124]: hmm not sure to understand. Currently it is working with "-" (1 dash) I simply updated it to match reality.

File: INSTALLATION.md (hunk 1 above)
Comment (SlnPons) [reply INSTALLATION.md]: Made some modifications, thanks :)

File: INSTALLATION.md [hunk 3]
```diff
@@ -179,20 +179,15 @@ If you want to get down to business, you can start your [Massalian journey](http
...
-Note that you must execute this script as an administrator in your terminal. To do so:
-1. Open Windows search bar 
-2. Type "cmd" 
-3. Right click on the "Open command prompt" icon and click on "Execute as an administrator"
-
-Now, copy the cmd below and paste it in the terminal you just opened.
+Follow the link below and download the file named "ThyraApp_windows-amd64.exe" on your computer. Then: 
+1. Open it by double clicking
+2. The script starts and here is what it does:
+  * If you have Thyra already installed on your computer, it install Thyra's icon tray.
```
Comment (gregLibert) [inline INSTALLATION.md:185]:
```suggestion
  * If you have Thyra already installed on your computer, it installs Thyra's icon tray.
```

**Review by gregLibert — CHANGES_REQUESTED** [review]
:mag: typo

**Review by gregLibert — APPROVED** [review]
:+1:

**Review by 0xMazout — APPROVED** [review]
Seems good !

File: README.md [hunk 4]
```diff
@@ -121,7 +121,7 @@ Accepts a URL, an IP address or one of the following values :
 - LOCALHOST : Expect Massa's network to be hosted at 127.0.0.1
 
 To use this option with a constant, you have to execute :
-`thyra-server node-server=LABNET`
+`thyra-server --node-server=LABNET`
 To use this option with a custom IP address, you have to execute :
 `thyra-server node-server=192.168.X.X`
```
Comment (peterjah) [inline README.md:126]: two dash also missing here ? !

**Pull request merged**